src/geefetch/data/get.py

```diff
@@ -142,6 +153,9 @@ def download_chip(
             **kwargs,
         )
         log.debug(f"Succesfully downloaded chip to [cyan]{out}[/]")
+    except ValueError as e:
+        log.error(f"ValueError in download_chip for {out}: {e}")
+        raise DownloadError from e
```

Comment:
Same comment as earlier, is it really necessary to show ValueError in the log when it is already contained in the exception `e` ?